integration-tests/environments/react-native/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "preinstall": "npm install ../../realm-integration-tests.tgz ../../realm.tgz --no-save",
+    "preinstall": "npm install ../../realm-integration-tests.tgz && npm install ../../realm.tgz --no-save --ignore-scripts",
```

Comment:
Can you comment on this change?

Reply:
```suggestion
    "preinstall": "npm install ../../realm-integration-tests.tgz ../../realm.tgz --no-save --ignore-scripts",
```

Reply:
There was an issue when installing `realm.tgz` if the package.json change is changed. Then it tries to build locally which is not possible on the current docker image. Since there is no need to do that I changed the install of the `realm.tgz` package to skip node build. 
If you think that `realm-integration-tests.tgz` should also be installed with --ignore-scripts I can change that

Reply:
ah I got what you mean. will fix it

Reply:
just comment on the --ignore-scripts for `realm-integration-tests.tgz`

Reply:
ok the --no-save is fixed below

Reply:
> Why do we need to perform the install over two runs of npm install?

This is because the change should minimize the impact. In other words I am not sure I want to disable scripts on the `realm-integration-tests` package. The issue is only that needs fixing is for the `realm` one only

Reply:
I wouldn't worry too much about that. There are currently no "prepublish" scripts that needs to run when the integration tests are installed into the environment. I still don't see a sufficient need to split this into two, with the downside of having to wait for two installs.

Reply:
fixed below